Makefile

```
@@ -1,45 +1,141 @@
-	@echo "  make install  - Install the package in development mode"
-	@echo "  make test     - Run tests with coverage"
-	@echo "  make lint     - Run linting and code formatting"
-	@echo "  make run      - Run the MCP server"
-	@echo "  make publish  - Build and publish the package to PyPI"
-	@echo "  make clean    - Clean build artifacts"
-	@echo "  make vulture  - Check for unused code with Vulture"
-	@echo "  make all      - Run install, lint, and test (default)"
-	@echo "  make help     - Show this help message" 
\ No newline at end of file
+# MCP DuckDuckGo - Development Makefile
+.PHONY: help install install-dev test lint format type-check security clean build publish pre-commit setup-dev
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)